templates/common/infra/bicep/core/security/keyvault-secret.bicep

```diff
@@ -1,3 +1,4 @@
+metadata description = 'Creates a secret or update a secret in a Key Vault.'
```

Comment:
```suggestion
metadata description = 'Creates or updates a secret in a Key Vault.'
```

Reply:
Should be "an Azure Key Vault" instead of "a Key Vault"